scripts/auth_collections.sh

```diff
@@ -8,7 +8,7 @@
 #    variable $TOK
 #       An admin token obtained from `get_admin_token` or `chef-automate iam token create admin-token-1 --admin`
 #    variable $TARGET_HOST
-#       A host running A2, either `https://a2-dev.test` or a remote machine.
+#       A host running Automate, either `https://a2-dev.test` or a remote machine.
```

Comment:
PR bonus: better to refer to the system as Automate instead of A2 nowadays.

Reply:
Thank you!